src/ripple/app/misc/NetworkOPs.cpp

```diff
@@ -2270,7 +2270,7 @@ NetworkOPsImp::getTxsAccountB(
                      std::string const& status,
                      Blob const& rawTxn,
                      Blob const& rawMeta) {
-        ret.emplace_back(std::move(rawTxn), std::move(rawMeta), ledgerIndex);
+        ret.emplace_back(rawTxn, rawMeta, ledgerIndex);
```

Comment:
@seelabs Why are these `move`s being removed? A `Blob` seems to be a great candidate for `move`.

Reply:
(I know you know this, but leaving a comment for future searches)
The parameters were `const`, so they won't be moved from.